Off-Top-App/off-top-python
# Ot 205 | Creating User Session

**What I Did:**
- Split up the old sessions api to post a new session as well as get all session from mongodb
- Changed a few fields from the Session object
- renamed src/ folder to backend-services/

**How To Test:**
- You can use postman or your browser and visit `http://localhost:5000/get-all-sessions ` to see a list of JSON objects appear
